assets/components/pages/event-list/event-list.twig

```diff
@@ -0,0 +1,44 @@
+{% set isHomepage = true %}
+
+{% extends "@templates/base/base.twig" %}
+
+{% block header %}
+  {% include '@organisms/header/header-light.twig' %}
+{% endblock %}
+
+{% block content %}
+
+<div class="container">
+  <h1>Événements</h1>
+</div>
+
+<div class="container mt-5">
+  <div class="row">
+    <div class="col-md-4">
+      <div>
+        {% include '@molecules/datepicker/datepicker.twig' %}
+      </div>
+      <div class="mt-5">
+        {% include '@molecules/filters/filters.twig' %}
+      </div>
+    </div>
+    <div class="col-md-8">
+      {% for i in 1..8 %}
+        {% include '@content-types/event/event.twig' %}
```

Comment:
Not super sexy those big fat teasers 😅 Need design review and solution on this one